opsdroid/events.py

```diff
@@ -133,6 +136,7 @@ def __init__(
 
         # Inherit the user, target and event from the event we are responding
         # to if they are not explicitly provided by this Event
+        event.user_id = event.user_id or self.user_id
```

Comment:
This suggestion, (with moving  the event.user line) will mean that we always populate `Event.user_id`. I think this is a good plan as it means it can always be relied on and can be overridden where appropriate.

```suggestion
        event.user = event.user or self.user
        event.user_id = event.user_id or self.user_id or event.user
```